baresipy/__init__.py

```
@@ -445,6 +464,9 @@ def run(self):
                 pass
             except KeyboardInterrupt:
                 self.running = False
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)